utils/game-utils.js

```diff
@@ -1,10 +1,3 @@
-const getPlayerPnumByUsername = (players, username) => {
```

Comment:
https://github.com/search?q=repo%3Acuttle-cards%2Fcuttle+getPlayerPnumByUsername&type=code

Not used anywhere.